packages/lib-classifier/src/plugins/drawingTools/experimental/components/TranscriptionLine/components/TranscriptionLineMark/TranscriptionLineMark.js

```diff
@@ -38,10 +38,9 @@ const TranscriptionLineMark = forwardRef((props, ref) => {
       fill={color}
       ref={ref}
       stroke={color}
-      strokeWidth={2}
     >
-      <line x1={x1 + offsetX} y1={y1 + offsetY} x2={x2} y2={y2} />
-      <line x1={x1} y1={y1} x2={x2} y2={y2} strokeWidth={GRAB_STROKE_WIDTH / scale} strokeOpacity='0' />
+      <line x1={x1 + offsetX} y1={y1 + offsetY} x2={x2} y2={y2} vectorEffect={'non-scaling-stroke'} />
+      <line x1={x1} y1={y1} x2={x2} y2={y2} strokeWidth={GRAB_STROKE_WIDTH} strokeOpacity='0' vectorEffect={'non-scaling-stroke'} />
```

Comment:
Drawing a new transcription line on this branch gives it stroke = GRAB_STROKE_WIDTH, which is 12. Is that intentional?

This branch:
<img width="357" alt="Screenshot 2024-06-26 at 3 21 34 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/a4140ae1-0260-45a8-b4ab-b2bf355804c8">

Production:
<img width="354" alt="Screenshot 2024-06-26 at 3 21 49 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/374f1cb5-5f2a-4997-811c-174dc875dc15">

Reply:
The line with GRAB_STROKE_WIDTH has opacity 0, so that’s fine. 

It does like the default stroke width is quite thick here. The visible line is the first line in the code (line 42.)

Line 43 is an invisible line that captures pointer events.

Reply:
I think the visible stroke width is 2px here.

Reply:
I've updated transcription lines to explicitly set a stroke width of 2px.

Reply:
Unlike all the other drawing tools, transcription lines aren’t composed from `Mark` components. That’s odd.

Reply:
For consistency with the other drawing tools, I’d recommend wrapping transcription lines in `Mark`, so that they are composed from the same base settings as other tools. I haven’t done that here as it might be a breaking change for transcription workflows.

Reply:
All the other drawing tools are wrapped in `Mark` here:
https://github.com/zooniverse/front-end-monorepo/blob/68a3e2bee5b6eb24012037fd6351e2ac1512f638/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.js#L78-L117